src/timeOperations.cpp

```diff
@@ -0,0 +1,114 @@
+// timeOperations namespace functions
+#include "folf.hpp"
+#include <iostream>
+#include <chrono>
+
+#if defined(_WIN32)
+
+#include <windows.h>
```

Comment:
pls remove codeless line (just for cleanness^^)

Reply:
yee i will, that was CLion, because of the ifdev, dunno why he does this